Latest clang is causing build failures because -Werror is used:

  edk2/MdeModulePkg/Include/Guid/ExtendedFirmwarePerformance.h:78:47:
  error: field Guid within 'FPDT_GUID_EVENT_RECORD' is less aligned than 'EFI_GUID'
  and is usually due to 'FPDT_GUID_EVENT_RECORD' being packed, which can lead to
  unaligned accesses [-Werror,-Wunaligned-access]

This has been reported upstream[1] so until this is resolved, ignore the warnings.

[1] https://edk2.groups.io/g/devel/message/86838

Upstream-Status: Pending
Signed-off-by: Ross Burton <ross.burton@arm.com>

diff --git a/BaseTools/Conf/tools_def.template b/BaseTools/Conf/tools_def.template
index f2bb6247e8..ca2b449f0a 100755
--- a/BaseTools/Conf/tools_def.template
+++ b/BaseTools/Conf/tools_def.template
@@ -2570 +2570 @@ DEFINE CLANG38_X64_TARGET           = -target x86_64-pc-linux-gnu
-DEFINE CLANG38_WARNING_OVERRIDES    = -Wno-parentheses-equality -Wno-tautological-compare -Wno-tautological-constant-out-of-range-compare -Wno-empty-body -Wno-unused-const-variable -Wno-varargs -Wno-unknown-warning-option -Wno-unused-but-set-variable -Wno-unused-const-variable
+DEFINE CLANG38_WARNING_OVERRIDES    = -Wno-parentheses-equality -Wno-tautological-compare -Wno-tautological-constant-out-of-range-compare -Wno-empty-body -Wno-unused-const-variable -Wno-varargs -Wno-unknown-warning-option -Wno-unused-but-set-variable -Wno-unused-const-variable -Wno-error=unaligned-access
